_includes/head.html

```diff
@@ -50,4 +50,8 @@
     ga('create', 'UA-73726852-1', 'auto');
     ga('send', 'pageview');
   </script>
+  <script>
+    var github_repo = "{{ site.repository | escape }}";
```

Comment:
Do these have to be defined here? 

They seem to only be used in github.js so wouldn't it make more sense for them to also be defined there?